packages/api/test/models/queryTemplate.test.ts

```diff
@@ -0,0 +1,307 @@
+import { beforeEach } from "mocha";
+import { QueryTemplateModel } from "@hypertool/common";
+import { assert } from "chai";
+import { assertThrowsAsync, queryTemplates } from "../helper";
+
+describe("QueryTemplate Model", function () {
```

Comment:
`QueryTemplate model`

The message that you specify to `describe` and the message that you specify to `it` are combined to form a sentence in the output. Keep that in mind.